lib/pronto/rubocop/patch_cop.rb

```diff
@@ -31,7 +31,14 @@ def processed_source
       end
 
       def registry
-        @registry ||= ::RuboCop::Cop::Registry.new(RuboCop::Cop::Cop.all)
+        @registry ||= ::RuboCop::Cop::Registry.new(
+          # keep support of older Rubocop versions
+          if RuboCop::Cop.const_defined?(:Registry) && RuboCop::Cop::Registry.respond_to?(:all)
+            RuboCop::Cop::Registry.all
+          else
+            RuboCop::Cop::Cop.all
+          end
```

Comment:
@ashkulz  sure, it would look nicer if we extract this into `all_cops` method, for example. should I go on and make this update? just checking because you already approved this PR.

Reply:
@ashkulz Extracted this into `all_cops`